hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/OzoneClientUtils.java

```diff
@@ -64,4 +69,38 @@ public static BucketLayout resolveLinkBucketLayout(OzoneBucket bucket,
     }
     return bucket.getBucketLayout();
   }
+
+  public static ReplicationConfig validateAndGetClientReplicationConfig(
```

Comment:
Should we have a unit test for the logic in this method, as there are quite a few IF statements etc.